unlock-app/src/hooks/useTransferFee.ts

```diff
@@ -29,3 +30,27 @@ export const useTransferFee = ({ lockAddress, network }: TransferFeeProps) => {
     getTransferFeeBasisPoints,
   }
 }
+
+export const useFetchTransferFee = ({
+  lockAddress,
+  network,
+}: TransferFeeProps) => {
+  const web3service = useWeb3Service()
+
+  async function fetchTransferFees() {
```

Comment:
I see there is a `getTransferFeeBasisPoints` above in that file. Can you DRY things up a bit please?